data/kaushalnitb.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Kaushal Kishor",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am a DevOps engineer"
```

There is a missing comma `,` which should be added in the end. 
```suggestion
  "bio": "I am a DevOps engineer",
```